src/azul/base_config.py

```diff
@@ -31,4 +31,4 @@ def entities(self) -> Iterable[str]:
     def index_names(self) -> Iterable[str]:
         entities = self.entities
         environment = os.getenv("STAGE_ENVIRONMENT", "dev")
-        return ["browser_{}_{}".format(entity, environment) for entity in entities]
+        return ["azul_{}_{}".format(entity, environment) for entity in entities]
```

Comment:
We don't use STAGE_ENVIRONMENT anymore. All references to that variable need to go.

Additionally, we want config.es_index_name() to be the only place where index names are generated. Duplicating that logic anywhere else is beside the point of the PR.